src/commands/Moderation/dehoist.ts

```diff
@@ -46,6 +53,13 @@ export default class extends SkyraCommand {
 			}
 
 			++counter;
+
+			// update the counter every 10 dehoists
+			if ((i + 1) % 10 === 0) {
+				const dehoistPercentage = (i / hoistedMembers.length) * 100;
+				await message.sendLocale('commandDehoistProgress', [{ count: i + 1, percentage: Math.round(dehoistPercentage) }]);
+				await sleep(5000);
```

Comment:
Any reason for this random sleep? In a real world scenario, the message would be sent every 30 seconds or so, nickname changes are pretty ratelimited.